src/backend/checkVersions.js

```diff
@@ -1,11 +1,6 @@
 import genUuid from "uuid/v1";
 
-let uuid = localStorage.getItem("uuid");
-
-if (!uuid) {
-  uuid = genUuid();
-  localStorage.setItem("uuid", uuid);
-}
+let uuid = genUuid();
 
 export const checkVersions = (hook, bridge) => {
```

Comment:
> The main question I have is uuid in check versions is used for? This code is not easy to implement in React Native, since it doesn't have localStorage but instead asyncStorage which is, well, async.

It seems like `checkVersions` ultimately does an async `fetch` call, so perhaps we could store the `uuid` in an abstract async local storage (backed by sync `localStorage` in browses but async storage in RN), and take advantage of the already-async behavior of `checkVersions`?

Reply:
Great idea @benjamn, I will implement it today!